tests/test_soup_searcher.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+from amz_review_scraper.soup_searcher import find_attribute
+import codecs
+import bs4
+import os
+
+my_path = os.path.abspath(os.path.dirname(__file__))
+path = os.path.join(my_path, "materials/output_file.html")
+
+raw_html = codecs.open(path,'r')
+soup = bs4.BeautifulSoup(raw_html, "lxml")
+product_json = {}
+
+def test_find_attribute():
```

Comment:
curious if any of these cases cover the `span` use case I see in the method.

Reply:
the third test uses the `span` use case